response_operations_ui/templates/collection_exercise/ce-sample-ci-section.html

```diff
@@ -1,66 +1,88 @@
-                "ths": [
-                    { 
-                        "value": "Sample & Collection instruments (CI)",
-                        "thClasses": "ons-u-fs-r--b"
-                    },
-                    {
-                        "value": "",
-                        "thClasses": "ons-u-ta"
-                    },
-                    {
-                        "value": "Uploaded " + '<em>' + sample.ingestDateTime[:-8] + '</em>' + sample.ingestDateTime[-8:],
-                        "thClasses": "ons-u-fs-r",
-                        "numeric": "true"
-                    },
-                ],
-                
-                "trs": [
-                    {
-                        "tds": [
-                            {
-                                "value": "Total businesses",
-                                "tdClasses": "ons-u-ta"
-                            },
-                            {
-                                "value": sample.totalSampleUnits,
-                                "name": "total-instruments",
-                                "tdClasses": "ons-u-ta-right"
-                            },
-                            {
-                                "value": ' ',
-                                "name": "upload-sample-time",
-                            }
-                        ]
-                    },
-                    {
-                        "tds": [
-                            {
-                                "value": "Collection instruments",
-                                "tdClasses": "ons-u-ta"
-                            },
-                            {
-                                "value": sample.expectedCollectionInstruments,
-                                "name": "total-instruments",
-                                "tdClasses": "ons-u-ta-right"
-                            },
-                            {
-                                "value": ' ',
-                                "name": "upload-sample-time",
-                            }
-                        ]
-                    }
-                ]
-            })
-        }}
+        {% if 'ACTIVE' == sample.state %}
+            {{
+                onsTable({
+                    "variants": ['compact', 'responsive'],
+                    "id": "sample-table",
+                    "caption": 'Loaded sample summary',
+                    "hideCaption": true,
+                    "ths": [
+                        { 
+                            "value": "Sample & Collection instruments (CI)",
+                            "thClasses": "ons-u-fs-r--b"
+                        },
+                        {
+                            "value": "",
+                            "thClasses": "ons-u-ta"
+                        },
+                        {
+                            "value": "Uploaded " + '<em>' + sample.ingestDateTime[:-8] + '</em>' + sample.ingestDateTime[-8:],
```

Comment:
I am guessing this is the problem, a None sample.ingestDateTime? Or is it something else

Reply:
Yeah this was the problem, it didn't have the `sample.state == "ACTIVE"` check so it was trying to subscript a None since the sample.ingestDateTime was None at the time (still in INIT phase)